polaris-react/src/components/Avatar/Avatar.tsx

```diff
@@ -153,12 +168,33 @@ export function Avatar({
   // Use `dominant-baseline: central` instead of `dy` when Edge supports it.
   const verticalOffset = '0.35em';
 
-  const avatarBody =
-    customer || !initials ? (
+  const avatarPath = polarisSummerEditions2023 ? (
+    <>
```

Comment:
Looks great! Only suggestion would be to maybe move these out of the component so there is less clutter inside the function logic. Not blocking though 👍 

```
const avatarPath = polarisSummerEditions2023
  ? avatarPathNext
  : avatarPathLegacy

// outside component
const avatarPathNext = <path ...
const avatarPathLegacy = <path ...
```

Reply:
The se23 hook needs to be inside the function though?

Reply:
> The se23 hook needs to be inside the function though?

Yes, sorry the code above wasn't very clear 😅 

```jsx
export const Avatar = (props) => {
  //other code

  const avatarPath = polarisSummerEditions2023
    ? avatarPathNext
    : avatarPathLegacy

  // other code
}

/* Outside component */
const avatarPathNext = <path ...
const avatarPathLegacy = <path ...